beetmoverscript/script.py

```diff
@@ -69,15 +76,17 @@
                                       manifest['mapping'][locale][deliverable]['s3_key'])
             dest_latest = os.path.join(manifest["s3_prefix_latest"],
                                        manifest['mapping'][locale][deliverable]['s3_key'])
+            balrog_manifest = manifest['mapping'][locale][deliverable].get('update_balrog_manifest')
```

Comment:
chapeau bas for this cool idea. way more neater to flag them from the manifest, rather than add more logic and if-checks for which artifacts to add in the `manifest.json`. Smart move, good job!